src/main/java/uk/gov/justice/hmpps/prison/repository/OffenderTransactionHistoryRepository.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.justice.hmpps.prison.repository;
+
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+import uk.gov.justice.hmpps.prison.repository.jpa.model.OffenderTransactionHistory;
+
+import java.time.LocalDate;
+import java.util.List;
+
+public interface OffenderTransactionHistoryRepository extends CrudRepository<OffenderTransactionHistory, String> {
```

Comment:
I missed this before, but should this be:
`CrudRepository<OffenderTransactionHistory, PK> `